### PR TITLE
Satisfy Kubernetes convention to only have alphanumeric characters and "-" in job names

### PR DESCRIPTION
**Pull request by Munsio**

### CODE REVIEW

File: cmd/eval-dev-quality/cmd/evaluate.go [hunk 1]
```diff
@@ -663,6 +664,9 @@ func (command *Evaluate) evaluateDocker(ctx *evaluate.Context) (err error) {
 
 // evaluateKubernetes executes the evaluation for each model inside a kubernetes run container.
 func (command *Evaluate) evaluateKubernetes(ctx *evaluate.Context) (err error) {
+	// Define a regex to replace all non alphanumeric characters and "-".
+	kubeNameRegex := regexp.MustCompile(`[^a-zA-Z0-9 -]+`)
```
Comment (bauersimon) [inline cmd/eval-dev-quality/cmd/evaluate.go:668]: I think we don't want the whitespace in the regex.

```sh
[^a-zA-Z0-9 -]+
[^a-zA-Z0-9-]+
```

Comment (Munsio) [reply cmd/eval-dev-quality/cmd/evaluate.go]: good catch

File: cmd/eval-dev-quality/cmd/evaluate.go [hunk 2]
```diff
@@ -746,6 +746,7 @@ func (command *Evaluate) evaluateKubernetes(ctx *evaluate.Context) (err error) {
 				Command: []string{
 					"kubectl",
 					"wait",
+					"--timeout", "24h",
```
Comment (bauersimon) [inline cmd/eval-dev-quality/cmd/evaluate.go:749]: Is that the whole evaluation or just a single container? I think a single container might be faster... But what if the full evaluation is longer? Would be a bit sad for someone running a huge eval and then it gets killed.

Comment (Munsio) [reply cmd/eval-dev-quality/cmd/evaluate.go:749]: It is a single container

**Pull request merged**